src/Console/Commands/OpenApiCommand.php

```diff
@@ -62,7 +62,7 @@ class OpenApiCommand extends QtCommand
      * Path to public debug bar resources
      * @var string
      */
-    private $publicOpenApiFolderPath = 'public/assets/OpenApiUi';
+    private $publicOpenApiFolderPath = 'public/assets/OpenApiUI';
```

Comment:
Change all this back to OpenApiUi instead of OpenApiUI nad correct all the places where it arise a problem